### PR TITLE
Google pubsub

### PR DESCRIPTION
**Pull request by timvancann**
**Description:**

Guide for the Home Assistant Google Pub/Sub component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20049

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/google_pubsub.markdown
Comment (frenck) [inline source/_components/google_pubsub.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
@timvancann Thanks for the PR! Left you one comment, could you please take a look? 👍

File: source/_components/google_pubsub.markdown
Comment (timvancann) [reply source/_components/google_pubsub.markdown]: Removed the `filter` from the example and added a note explaining why users should care about this setting 👍 .

**Review by frenck — DISMISSED** [review]
Thanks for updating the PR @timvancann! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by timvancann** [comment]
@frenck Version has been bumped to `0.88`. Still waiting for approval of home-assistant/home-assistant#20049.

File: source/_components/google_pubsub.markdown
Comment (klaasnicolaas) [inline source/_components/google_pubsub.markdown]: This is not a correct category.
Once you have adjusted this, the PR can be merged.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**
